policies/constant.go

```diff
@@ -0,0 +1,7 @@
+package policies
+
+import "time"
+
+func GetConstantPolicy(duration time.Duration, max int) Policy {
+	return GetExponentialPolicy(1, duration, max)
```

Comment:
if it's really a constant, what would the max do? why not just write the constant policy?

Reply:
let's just write the constant policy that backs off by duration time.